File: ChallengeAlpha/app/src/main/java/com/hurb/challengealpha/model/Tax.kt
package com.hurb.challengealpha.model

import com.google.gson.annotations.SerializedName

data class Tax(

    @SerializedName("type") val type: String,
    @SerializedName("name") val name: String,
    @SerializedName("amount") val amount: Double,
    @SerializedName("amount_original") val amount_original: Double,
    @SerializedName("currency") val currency: String,
    @SerializedName("currency_original") val currency_original: String
)

File: DesafioHurb/Domain/Model/Tax.swift
//
//  Tax.swift
//  DesafioHurb
//
//  Created by Edson Aparecido Guido on 13/09/20.
//  Copyright © 2020 Edson Aparecido Guido. All rights reserved.
//

import Foundation

struct Tax: Codable {
    let type: String
    let name: String
    let amount: Double
    let amountOriginal: Double
    let currency: String
    let currencyOriginal: String

    enum CodingKeys: String, CodingKey {
        case type, name, amount
        case amountOriginal = "amount_original"
        case currency
        case currencyOriginal = "currency_original"
    }
}
